deeds-dapp-service/src/main/java/io/meeds/deeds/web/filter/RequestDispatcherFilter.java

```diff
@@ -189,7 +205,7 @@ private String getPageHeaderMetadataContent(HttpServletRequest request, String s
 
   private String getLanguage(HttpServletRequest request) {
     String path = request.getServletPath();
-    return DAPP_PATHS_FR.contains(path) || STATIC_PATHS_FR.contains(path) ? "fr" : "en";
+    return path.contains("/fr") ? "fr" : "en";
```

Comment:
```suggestion
    return path.startsWith("/fr/") ? "fr" : "en";
```

Reply:
changed! with  path.startsWith("/fr") (without "/" at the end) , for the case of  home page in fr

Reply:
ok, then it should be like this, to be more consistent in fact

```suggestion
    return StringUtils.equals(path, "/fr") || path.startsWith("/fr/") ? "fr" : "en";
```

Reply:
done!